pkg/env/env.go

```diff
@@ -85,8 +87,9 @@ func (p *processor) RenderAll(host, dir string) error {
 		if err != nil {
 			return fmt.Errorf("failed to create template: %s %v", path, err)
 		}
-		if err := t.Execute(writer, p.getHostEnv(host)); err != nil {
-			return fmt.Errorf("failed to render env template: %s %v", path, err)
+		env := p.getHostEnv(host)
+		if err := t.Execute(writer, env); err != nil {
+			return fmt.Errorf("failed to render env %v into template: %s %v", env, path, err)
```

Comment:
return fmt.Errorf("failed to render env(%v) into template at path(%s): %v", env, path, err)